### PR TITLE
Update the link for teachers in Wales

### PR DESCRIPTION
**Pull request by peteryates**
The site at the old link appears to have broken; it's just a Plesk admin login and a certificate error.

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -37,7 +37,7 @@
           <p>Find teacher training options if you live in:</p>
           <ul class="govuk-list govuk-list--bullet">
             <li><a href="https://teachinscotland.scot">Scotland</a></li>
-            <li><a href="https://www.discoverteaching.wales/routes-into-teaching/">Wales</a></li>
+            <li><a href="https://gov.wales/becoming-teacher">Wales</a></li>
```
Comment (gemmadallmandfe) [inline app/views/pages/home.html.erb:40]: Suggest we link to `https://educators.wales/` instead as this is the Wales link used on GIT from 

* https://getintoteaching.education.gov.uk/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk
* https://getintoteaching.education.gov.uk/train-to-teach-in-england-as-an-international-student

Comment (gemmadallmandfe) [reply app/views/pages/home.html.erb:40]: And is also the site that the discoverteaching.wales link redirects to

Comment (peteryates) [inline app/views/pages/home.html.erb:40]:
```suggestion
            <li><a href="https://educators.wales/">Wales</a></li>
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**